melodic/distribution.yaml

```diff
@@ -2001,6 +2001,22 @@ repositories:
       url: https://github.com/ros/pluginlib.git
       version: melodic-devel
     status: maintained
+  pointcloud_to_laserscan:
+    doc:
+      type: git
+      url: https://github.com/ros-perception/pointcloud_to_laserscan.git
+      version: indigo-devel
```

Comment:
Are you sure about using `indigo-devel` here and for `source`? The repo does have a `lunar-devel` branch.

Reply:
Great catch, thank you